kitsune/users/urls.py

```diff
@@ -98,6 +98,6 @@
             name='users.fxa_authentication_init'),
         url(r'^fxa/logout/$', never_cache(views.FXALogoutView.as_view()),
             name='users.fxa_logout_url'),
-        url(r'^fxa/events/$', never_cache(views.WebhookView.as_view()), name='users.fxa_webhook'),
+        url(r'^fxa/events', never_cache(views.WebhookView.as_view()), name='users.fxa_webhook'),
```

Comment:
why do we need this change?

Reply:
in bug 1595529 you requested `support-dev.allizom.org/fxa/events` as the webhook url

I'm realising now that this was regex... so I should probably change it to something like `^fxa/events/?$` to handle both with and without a trailing slash